### PR TITLE
wifi: Remove support for nrf7002 Rev-A

### PR DESCRIPTION
**Pull request by krish2718**

### CODE REVIEW

**Comment by krish2718** [comment]
@ioannisg FYI

**Review by sachinthegreen — APPROVED** [review]

**Comment by SeppoTakalo** [comment]
What is the story behind this?
LwM2M sample uses nRF7002 shield, is there any other shields than Rev A?

**Comment by krish2718** [comment]
> What is the story behind this? LwM2M sample uses nRF7002 shield, is there any other shields than Rev A?

Yes, we have Rev-B shields as well, a.k.a Rev-B EK. All Rev-A are being replaced with Rev-B for all internal teams.

**Review by SeppoTakalo — APPROVED** [review]

File: doc/nrf/releases/release-notes-changelog.rst
Comment (richabp) [inline doc/nrf/releases/release-notes-changelog.rst:450]: Can this be added only under one section instead of Protocols and Samples both?

Comment (krish2718) [reply doc/nrf/releases/release-notes-changelog.rst:450]: Ok, but which one?

Comment (krish2718) [reply doc/nrf/releases/release-notes-changelog.rst:450]: Actually IMHO it might be better to highlight in both, as we made changes in both areas.

**Comment by krish2718** [comment]
@krga2022 @bama-nordic need your approval before we merge this.

**Review by mjd90 — APPROVED** [review]
Great (apologies any remaining revA users, we need to move forward)

**Review by richabp — APPROVED** [review]

**Review by ioannisg — APPROVED** [review]
I am OK with removing RevA from NCS support

**Pull request merged**
